debian/patches/01-regolith-trawl.patch

```diff
@@ -49,25 +49,6 @@ Index: sway-regolith/meson.build
  wlroots_features = {
  	'xwayland': false,
  	'libinput_backend': false,
-@@ -248,7 +269,7 @@ configure_file(
- )
- 
- install_data(
--	'sway.desktop',
-+	'sway-regolith.desktop',
- 	install_dir: join_paths(datadir, 'wayland-sessions')
- )
- 
```

Comment:
We should remove this section from meson.build if we don't want to keep the base session accessible. 

```patch
-  install_data( 
-           'sway.desktop',
-           'sway-regolith.desktop',
-           install_dir: join_paths(datadir, 'wayland-sessions')
- )
```

Reply:
removed desktop session file as suggested, thanks